git_cookbook.md

```diff
@@ -22,7 +22,7 @@ function pr () {
 ```
 
 #### Convenience Alias Commands
-For use with your shell's `alias` command.
+For use with your shell's `alias` command, or you can copy and paste them into your ~/.bashrc file.
```

Comment:
In the previous section of this file, it recommends putting the the scripts inside ` ~/.bash_profile`. We should be consistent and stick to one of these